<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BHXServerClass</class>
 <widget class="QMainWindow" name="BHXServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BHXServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pb_startserver">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>391</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>启动服务器</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tb_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>581</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_ip">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ip:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_clean_log">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清理消息</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bhxserver.qrc"/>
 </resources>
 <connections/>
</ui>
